source/includes/api/_vocabulary.md

```diff
@@ -1,694 +1,1297 @@
+404            | 404 - {\"errors\":[{\"status\":404,\"detail\":\"Dataset with id `<dataset id>` doesn't exist\"}]}      | You are trying to clone a vocabularies for a source dataset that doesn't exist.
+
+## Deleting vocabularies/tags for a resource
+
+There are two types of endpoints for deleting vocabularies/tags: one allows you to delete a single vocabulary (and its
+tags) for a resource, while the other supports deleting all the vocabularies/tags for a single resource. Some use cases
+for these include deleting vocabularies prior to deleting the actual resource, or typical housekeeping tasks you may
+want to carry out on your data.
+
+### Deleting a single vocabulary/tags for a resource
+
+> Deleting a single vocabulary/tag for a dataset
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/dataset/<dataset-id>/widget/<widget-id>/vocabulary
+curl -X DELETE https://api.resourcewatch.org/v1/dataset/<dataset-id>/vocabulary/<vocabulary-id>
 ```
 
-> Getting Vocabularies related with a Layer
+> Deleting a single vocabulary/tag for a widget
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/dataset/<dataset-id>/layer/<layer-id>/vocabulary
+curl -X DELETE https://api.resourcewatch.org/v1/dataset/<dataset-id>/widget/<widget-id>/vocabulary/<vocabulary-id>
 ```
 
-> Example of request with real data
+> Deleting a single vocabulary/tag for a layer
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/dataset/942b3f38-9504-4273-af51-0440170ffc86/vocabulary
+curl -X DELETE https://api.resourcewatch.org/v1/dataset/<dataset-id>/layer/<layer-id>/vocabulary/<vocabulary-id>
 ```
 
-> Example of response with real data
+> Example response
 
 ```json
 {
-  "data": [
-    {
-      "id": "country",
-      "type": "vocabulary",
-      "attributes": {
-        "tags": [
-          "Spain",
-          "Italy",
-          "Portugal"
-        ]
-      }
-    },
-    {
-      "id": "color",
-      "type": "vocabulary",
-      "attributes": {
-        "tags": [
-          "red",
-          "green",
-          "blue"
-        ]
-      }
-    }
-  ]
+    "data": [
+        {
+            "id": "country",
+            "type": "vocabulary",
+            "attributes": {
+                "tags": [
+                    "Spain",
+                    "Italy",
+                    "Portugal"
+                ]
+            }
+        },
+        {
+            "id": "color",
+            "type": "vocabulary",
+            "attributes": {
+                "tags": [
+                    "red",
+                    "green",
+                    "blue"
+                ]
+            }
+        }
+    ]
 }
 ```
 
-## Getting a single relationship (broken now)
+This set of endpoints will allow you to delete a single vocabulary and its tags, for a resource.
 
-> Getting a single relationship between a Vocabulary and a Dataset
+As a query param, you can optionally provide an `app` or `application` values, to identify the application of the
```

Comment:
What will happen if you don't do this and there are two vocabularies with the same name but different applications (can that even happen?)? Will it potentially delete multiple? Will it pick one randomly?